hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -766,6 +766,12 @@ class Emit[C](
           }
         }
 
+      case Begin(xs) =>
+        assert(xs.length != 0)
+        xs.init.foreach(emitVoid(_))
+        assert(xs.last.pType != TVoid)
```

Comment:
A `pType` would never equal `TVoid`. Should be `PVoid`? Though I also wonder why it's not ok for `Begin` to end with a `Void` type. Do we mandate that all IR's return something in the end?